src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 2;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+
+  let sum = days * PRICE_PER_DAY;
+
+  if (days < LONG_TERM && days > SHORT_TERM) {
+    sum = days * PRICE_PER_DAY - SHORT_TERM_DISCOUNT;
+  } else if (days >= LONG_TERM) {
+    sum = days * PRICE_PER_DAY - LONG_TERM_DISCOUNT;
```

Comment:
There is no need to recalculate `days * PRICE_PER_DAY` in lines 16 and 18 since it's already calculated in line 13 as `sum`. You can simply subtract the discount from `sum` to avoid redundant calculations.